pipelines/pipelines/eclipse/build-branch.yaml

```diff
@@ -248,47 +248,6 @@ spec:
       - --health
 # 
```

Comment:
We'll also need to remove the `branch-docker-build-eclipse-p2` step in the pipeline to stop building the docker image for the p2 site

Reply:
Will be removed in a separate PR

Reply:
I will remove the` branch-docker-build-eclipse-p2` as part of the next PR which will come after I delete the docker images.